defaults/main.yml

```diff
@@ -1,10 +1,13 @@
+mysql_hardening_enabled: no
 # general configuration
 mysql_hardening_user: 'mysql'
+mysql_hardening_group: 'mysql'
```

Comment:
This should by default be `root`, so the tests pass.
If you think this is wrong, we should discuss this in the mysql-tests repository.